community/mg/checksums

```diff
@@ -0,0 +1 @@
+git  git+https://github.com/hboetes/mg.git
```

Comment:
A new release was tagged today. This could be used instead of tracking git: https://github.com/hboetes/mg/archive/20200215.tar.gz